app/src/main/java/com/tohami/newsapi/ui/news/NewsActivity.java

```diff
@@ -0,0 +1,108 @@
+package com.tohami.newsapi.ui.news;
+
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.os.PersistableBundle;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.widget.Toolbar;
+import androidx.drawerlayout.widget.DrawerLayout;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+
+import com.google.android.material.navigation.NavigationView;
+import com.tohami.newsapi.R;
+import com.tohami.newsapi.base.BaseActivity;
+
+public class NewsActivity extends BaseActivity {
```

Comment:
This activity should be renamed to MainActivity

Reply:
done